src/Entity/Post.php

```diff
@@ -3,23 +3,29 @@
 namespace App\Entity;
 
 use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
 use Symfony\Component\Validator\Constraints as Assert;
 
 /**
+ * @Gedmo\SoftDeleteable(fieldName="deletedAt", timeAware=false, hardDelete=false)
  * @ORM\Entity(repositoryClass="App\Repository\PostRepository")
  */
 class Post
 {
+    public const STATUS_PENDING = 0;
+    public const STATUS_ACTIVE = 1;
+    public const STATUS_DELETED = 2;
```

Comment:
Seems redundant with `deletedAt`

Reply:
As we need status "pending", I thought we should complete the status for every situation

Reply:
With `deletedAt` we already have status deleted info

Reply:
ok but when I delete a post is the status (in database) supposed to stay "active" or "pending"? I think it should be "deleted"